src/components/PageDefinitions.js

```diff
@@ -219,109 +227,98 @@ class PageDefinitions extends Component {
     this.refs.contributeModal.open()
   }
 
-  name = coordinates => {
+  name(coordinates) {
     return coordinates.name ? coordinates.name : null
   }
 
-  namespace = coordinates => {
+  namespace(coordinates) {
     return coordinates.namespace ? coordinates.namespace : null
   }
 
-  provider = coordinates => {
+  provider(coordinates) {
     return coordinates.provider ? coordinates.provider : null
   }
 
-  type = coordinates => {
+  type(coordinates) {
     return coordinates.type ? coordinates.type : null
   }
 
-  releaseDate = coordinates => {
+  releaseDate(coordinates) {
     const definition = this.props.definitions.entries[EntitySpec.fromCoordinates(coordinates).toPath()]
     const described = get(definition, 'described')
     if (described && described.releaseDate) return described.releaseDate
```

Comment:
`return get(definition, 'described.releaseDate')`

The one quirk is you have make sure that both `null` and `undefined` are reasonable return results

See also the `license` function